server/src/main/java/org/elasticsearch/cluster/metadata/DataStream.java

```diff
@@ -273,6 +273,13 @@ public IndexMode getIndexMode() {
     public DataStream rollover(Index writeIndex, long generation) {
         ensureNotReplicated();
 
+        return unsafeRollover(writeIndex, generation);
+    }
+
+    /**
+     * Like #rollover, but does no validation, use with care only.
```

Comment:
Maybe use `{@link #rollover(Index, long)}` instead of `#rollover` in java docs?

Reply:
Thanks for spotting that, fixed in da77aaee1931f03c82ddfcd50ec93fc86bd00d9d